src/main/scala/tilelink/Broadcast.scala

```diff
@@ -74,72 +173,143 @@ class TLBroadcast(params: TLBroadcastParams)(implicit p: Parameters) extends Laz
     }
   )
 
+  /** Interrupt Node for filter. Each Filter has a interrupt. */
   val intNode = if (params.control.isEmpty) None else Some(IntSourceNode(IntSourcePortSimple(num = 1)))
+  /** Control Node for filter.
+    * [[ProbeFilter.useRegFields]] can be defined to add register to control.
+    */
   val controlNode = params.control.map(x => TLRegisterNode(
     address   = Seq(x.address),
     beatBytes = x.beatBytes,
     device    = new SimpleDevice("cache-controller", Seq("sifive,broadcast0"))))
 
   lazy val module = new LazyModuleImp(this) {
+    /* Number of inward node will decide the number bank. */
```

Comment:
This statement is true only for the generic topology that we use by default. You could design a network using `_ :*= bh :*= _` instead of `_ :=* bh :=* _` where the  outward side decides how many banks there are.

Reply:
Got it. I'll make it more clear.

Reply:
Put another way, this node is just an adapter node. The number of banks is the number of edges going through it, and the number of inward edges must equal the number of outwards edges. Either direction could be made to pick how many edges there are.